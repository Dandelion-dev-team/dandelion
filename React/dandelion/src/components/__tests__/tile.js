import React from "react"
import renderer from "react-test-renderer"
import Tile from "../tile"


describe("Tile", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(<Tile name="Map" tile_image="img" tile_color="#e3c3ca" link="/map"/>)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})

