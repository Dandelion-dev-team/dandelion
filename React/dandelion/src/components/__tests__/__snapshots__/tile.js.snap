// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile renders correctly 1`] = `
<tile>
  <div
    className="tile_div"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "#e3c3ca",
      }
    }
  >
    <img
      src="img"
    />
  </div>
</tile>
`;
